batch/batch/worker/worker.py

```diff
@@ -1496,14 +1501,12 @@ async def run(self):
                 with self.step('post-job finally block'):
                     if self.disk:
                         try:
-                            await self.disk.delete()
+                            await asyncio.wait_for(await CLOUD_WORKER_API.delete_disk(self.disk), DISK_OPERATION_TIMEOUT_SECONDS)
```

Comment:
wait_for should take an awaitable. As written, this code would wait for delete disk and then, I think, error because `wait_for` expects an awaitable.